scripts/setup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+dos2unix flake.nix
+"/root/.nix-profile/bin/nix" "develop" "--extra-experimental-features" "nix-command" "--extra-experimental-features" "flakes" "/dissolve#"
```

Comment:
```suggestion
"/root/.nix-profile/bin/nix" "develop" "/dissolve#"
```
Since we added the nix-command and flakes features to nix.conf in line six of the docker file, we shouldn't need to specify them in the script.